lib/teiserver_web/templates/admin/user/index.html.heex

```diff
@@ -1,5 +1,6 @@
 <% bsname = view_colour()
-is_moderator = allow?(@conn, "Moderator") %>
+is_moderator = allow?(@conn, "Moderator")
+is_system = allow?(@conn, "System") %>
```

Comment:
Why is this permission different than the `Server` you used elsewhere? I'm not on top of the permission system, so I may very well have missed something.

Reply:
I don't know what I originally thought here. I switched to ```is_server``` to have it the same way as the other.